cinema/serializers.py

```diff
@@ -1 +1,127 @@
 # write serializers here
+from rest_framework import serializers
+
+from cinema.models import Genre, CinemaHall, Actor, Movie, MovieSession, Order, Ticket
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = "__all__"
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CinemaHall
+        fields = [
+            "id",
+            "name",
+            "rows",
+            "seats_in_row",
+            "capacity",
+        ]
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    full_name = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Actor
+        fields = ["id", "first_name", "last_name", "full_name"]
+
+    def get_full_name(self, obj):
+        return f"{obj.first_name} {obj.last_name}"
+
+
+# class ActorListSerializer(serializers.ModelSerializer):
+#     class Meta:
+#         model = Actor
+#         fi
+class MovieSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Movie
+        fields = ["id", "title", "description", "duration", "genres", "actors"]
+
+
+class MovieListSerializer(serializers.ModelSerializer):
+    actors = serializers.SerializerMethodField()
+    genres = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Movie
+        fields = ["id", "title", "description", "duration", "genres", "actors"]
+
+    def get_actors(self, obj):
+        return [str(actor) for actor in obj.actors.all()]
+
+    def get_genres(self, obj):
+        return [str(genre) for genre in obj.genres.all()]
+
+
+class MovieRetrieveSerializer(serializers.ModelSerializer):
+    actors = ActorSerializer(many=True)
+    genres = GenreSerializer(many=True)
+
+    class Meta:
+        model = Movie
+        fields = ["id", "title", "description", "duration", "genres", "actors"]
+
+
+class MovieSessionSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = MovieSession
+        fields = "__all__"
+
+
+class MovieLSerializer(serializers.ModelSerializer):
+    actors = serializers.SerializerMethodField()
+    genres = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Movie
+        fields = ["id", "title", "description", "duration", "genres", "actors"]
+
+    def get_actors(self, obj):
+        return [str(actor) for actor in obj.actors.all()]
+
+    def get_genres(self, obj):
+        return [str(genre) for genre in obj.genres.all()]
+
```

Comment:
The `MovieLSerializer` class appears to be redundant as it duplicates the functionality of `MovieListSerializer`. Consider removing `MovieLSerializer` to avoid confusion and maintain cleaner code.